platform_main_feature.views_default.inc

```diff
@@ -315,6 +314,7 @@ function platform_main_feature_views_default_views() {
     t('‹ previous'),
     t('next ›'),
     t('last »'),
+    t('lcp_mods_facet_date_mdt'),
```

Comment:
... Was the label left enabled, and it is using the default (which is the field name)?